next/unittests/databaseapi_test.py

```diff
@@ -0,0 +1,250 @@
+import pytest
```

Comment:
The convention with py.test keep the tests alongside the file they're testing, and rename to 'test_*.py'. i.e., both 'databaseapi.py' and 'test_databaseapi.py' in the same directory.

If there are two test files, the tests can be in a sub directory 'tests'